lib/acidic_job/testing.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require "active_job/queue_adapters"
+require "active_job/base"
+require "active_job/test_helper"
+require "active_job/test_case"
+require "database_cleaner"
+
+module AcidicJob
+  module Testing
+    def self.included(mod)
+      mod.class_eval "self.use_transactional_tests = false if respond_to?(:use_transactional_tests)"
```

Comment:
Does `mod.use_transactional_tests = false if mod.respond_to?(:use_transactional_tests)` also work? I'd love to avoid the `eval`

Reply:
good material for a Ruby card :)

honestly I don't know off the top of my head, but I'll give it a try!

Reply:
no luck, sorry :-(

Reply:
Din what way? I can try to experiment with some other paths.

Reply:
I tried it out and got the transaction errors again... sure, please suit yourself!